src/app/editor/editor.js

```diff
@@ -99,8 +99,12 @@ class Editor {
     this.editor = ace.edit(el)
     ace.acequire('ace/ext/language_tools')
 
-    // Unmap ctrl-t & ctrl-f
-    this.editor.commands.bindKeys({ 'ctrl-t': null })
+    // Unmap ctrl-t & ctrl-f & ctrl-l & cmd-l
+    this.editor.commands.bindKeys({
+      'ctrl-t': null,
```

Comment:
could you please remove ctrl-t from the list. Currently, it's not being used by Ace editor. thanks

Reply:
@ckshei

Reply:
@LianaHus - Yup I can remove ctrl-t. On brace it's mapped as "transposeletters" not sure what that does but assuming that it's fine? I'll also update the comment 

```    name: "transposeletters",
    bindKey: bindKey("Ctrl-T", "Ctrl-T"),
    exec: function(editor) { editor.transposeLetters(); },
    multiSelectAction: function(editor) {editor.transposeSelections(1); },
    scrollIntoView: "cursor"```